pkg/polaris/graphql/infinity-k8s/infinity_k8s.go

```diff
@@ -83,6 +83,10 @@ type ExportK8sResourceSetSnapshotJobConfig struct {
 	Filter              string `json:"filter,omitempty"`
 }
 
+type BaseOnDemandSnapshotConfigInput struct {
```

Comment:
any reason behind this specific name? 
can't we just call this `OnDemandSnapshotJobConfig`?

Reply:
Borrowed it from the graphql schema name. It is shared among various components. Should I rename to your suggestion?

Reply:
Let's leave it to be the same as the graphql object name.